test/services/user/registration.test.js

```diff
@@ -0,0 +1,113 @@
+const { expect } = require('chai');
+
+const app = require('../../../src/app');
+const accountModel = require('../../../src/services/account/model');
+const { consentModel } = require('../../../src/services/consent/model');
+const { userModel, registrationPinModel } = require('../../../src/services/user/model');
+const { schoolModel } = require('../../../src/services/school/model');
+const testObjects = require('../helpers/testObjects')(app);
+
+const patchSchool = (system, schoolId) => schoolModel.findOneAndUpdate({ _id: schoolId }, {
+	$push: {
+		systems: system._id,
+	},
+}, { new: true }).lean().exec();
+
+const createAccount = (system) => accountModel.create({
+	lasttriedFailedLogin: '1970-01-01T00:00:00.000+0000',
+	activated: false,
+	username: 'fritz',
+	password: '',
+	systemId: system._id,
+});
+
+const createPin = (pin = 6716, email) => registrationPinModel.create({
+	verified: false,
+	email: email || `${Date.now()}@test.de`,
+	pin,
+});
+
+describe.only('registration', () => {
+	let account;
+	let pin;
+	let system;
+	let userId = '';
+	const { schoolId } = testObjects.options;
+
+	it('sso registration should work', async () => {	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5016272)